Feature/translate v1

cristofer-dev opened the pull request.

cristofer-dev committed:
ADD translate v1

cristofer-dev committed:
Merge pull request #46 from iv-devs/development

Translate

cristofer-dev committed:
ADD Text english

Xabadu commented on evento2018.html:
```diff
@@ -15,20 +15,21 @@
 
     <div class="col-md-6 col-sm-6 col-xs-6 text-center  m-t-10vh">
-      <h3 class="rojo">Organiza</h3>
+      <h3 class="rojo">Organize</h3>
```
`Organizers`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
   <div id="objetivo" class="row ">
     <div class="col-md-12  text-center ">
-      <h1>Nuestro Objetivo...</h1>
+      <h1>Our objective...</h1>
```
`Our goal...`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
-        positivamente a las personas y al entorno que nos rodea, más aún cuando esa capacidad de crear se suma a la de otros,
-        comprendiendo asi que el todo es más que
-        <b>la suma de las partes</b>
+        We are a Community that firmly believes that within each person there is an unparalleled potential to create and affect positively to the people and the environment that surrounds us, especially when that capacity to create adds to that of others, understanding that the whole is more than <b> the sum of the parts </b>
```
`affect positively the people and the environment that surrounds us`
`sum of its parts`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
-        <b>la programación</b>. Regularmente realizamos actividades que puedes revisar
-        <a href="http://ivdevs.com/evento">aquí</a> y anualmente organizamos una conferencia con múltiples charlas que buscan ser el
-        <b>update tecnológico</b> que permitan fortalecer tanto para desarrolladores locales como a entusiastas de la tecnología.
+        That moves us to do <b>things</b> that add value to our environment with what we do best ... programming. We regularly carry out activities that you can review <a href="http://ivdevs.com/evento">here</a> and annually we organize a conference with multiple talks that seek to be the technological update that allows strengthening both for local developers and technology enthusiasts.
```
`That pushes us to do`
`activities that you can view`
`and we organize a yearly conference`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
       <p>
         <span class="rojo bold">></span>
-        <a href="/conference2017/">Mira como estuvo la conferencia del 2017</a>
+        <a href="/conference2017/">See how the 2017 conference</a>
```
`See our 2017 conference`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
   <div id="numeros" class="row ">
     <div class="col-md-12  text-center ">
-      <h1>Nosotros...</h1>
+      <h1>US...</h1>
```
`About Us`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
       <span class="valor">26</span>
       <br>
-      <span class="textos">Meetups realizados</span>
+      <span class="textos">Meetups made</span>
```
`Previous meetups` (o events)

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
     <div class="col-md-6 col-sm-6 col-xs-12 text-center img">
       <img src="/assets/img/partners/logo_noders_white.png" alt="">
-      <span>La comunidad de NodeJS y JS en Lationamérica</span>
+      <span>The community of NodeJS y JS in Latin America</span>
```
`Node.js`
`The Most Electrifying Node.js community in Latin America`

Xabadu commented on evento2018.html:
```diff
@@ -37,120 +38,99 @@ <h3 class="rojo">Sponsor Sede</h3>
     <div class="col-md-6 col-sm-6 col-xs-12 text-center img">
       <img src="http://www.programadores.cl/images/proin-logo-min.png" alt="">
-      <span>Comunidad de Programadores e Informáticos de Chile</span>
+      <span>Community of Programmers of Chile</span>
```
`in Chile`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
-        <li>- Campañas de pago en Facebook.</li>
-        <li>- Sitio web.</li>
+        <li>- Social networks.</li>        
+        <li>- webpage.</li>
```
`Website`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
     </div>
     <div class="col-md-4">
-        <h3>Durante el evento</h3>
+        <h3>During Event</h3>
```
`During the event`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
-          <li>- Publicación de agradecimientos oficiales.</li>
-          <li>- Noticia en sitios Online involucrados.</li>
+          <li>- Publishing of the recorded sessions.</li>
+          <li>- Publishing of the photographic in social networks.</li>
```
`photos`
`social media`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
-        <li>- Menciones en redes sociales.</li>
-        <li>- Campañas de pago en Facebook.</li>
-        <li>- Sitio web.</li>
+        <li>- Social networks.</li>        
```
`Social media`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
   <div id="sponsor" class="row ">
     <div class="col-md-12  text-center ">
-      <h1 class="rojo">Se sponsor</h1>
+      <h1 class="rojo">Be sponsor</h1>
```
`Be a sponsor`

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
         <div class="row">
         <h3><strong>GOLD</strong></h3>
-        <h4>Desde $200.000</h4>
+        <h4>400 USD</h4>
```
Cuando el dolar estaba a 500 (?)

Xabadu commented on evento2018.html:
```diff
@@ -202,50 +182,49 @@ <h1 class="rojo">Sponsors Anteriores...</h1>
+        <h4>400 USD</h4>
           <ul>
-            <li>- Presentación del sponsor</li>
+            <li>- Logo in website with link to website</li>
```
`Logo in our Website linked to the sponsor's site`

cristofer-dev committed:
ADD img past speakers

 Changes to be committed:
	modified:   assets/css/main.css
	modified:   evento2018.html

cristofer-dev committed:
UPDATE English text

 Changes to be committed:
	modified:   evento2018.html

cristofer-dev committed:
UPDATE English text

 Changes to be committed:
	modified:   evento2018.html

cristofer-dev commented:
@Xabadu DONE!
